docs/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md

```diff
@@ -216,7 +216,11 @@ In addition to these default roles, the following Rancher project roles can be a
 |--------------------------|-------------------------------|-------|------|
 | View Monitoring* | [monitoring-ui-view](#additional-monitoring-clusterroles)    |    2.4.8+    |  9.4.204+ |
 
-\* A user bound to the **View Monitoring** Rancher role and read-only project permissions can't view links in the Monitoring UI. They can still access external monitoring UIs if provided links to those UIs. If you wish to grant access to users with the **View Monitoring** role and read-only project permissions, move the `cattle-monitoring-system` namespace into the project. 
+:::note
+
+A user bound to the **View Monitoring** Rancher role and read-only project permissions can't view links in the Monitoring UI. They can still access external monitoring UIs if provided links to those UIs. If you wish to grant access to users with the **View Monitoring** role and read-only project permissions, move the `cattle-monitoring-system` namespace into the project.
+
+:::
```

Comment:
How do you feel about also having a similar note in the top level section, like maybe in:
https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting#default-components-and-deployments?  Might make sense to add a section with an overview of the monitoring objects as well, and then add a warning like : "Hey, read project permissions don't necessarily apply to monitoring's obejcts. go see the RBAC section for more details"

Reply:
I likwe that a lot